src/frontends/pytorch/src/op/selu.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset8.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_selu(NodeContext& context) {
+    auto x = context.get_input(0);
+    auto alpha = context.mark_node(opset8::Constant::create(element::f64, Shape{}, {1.6732632423543772848170429916717}));
+    auto lambda = context.mark_node(opset8::Constant::create(element::f64, Shape{}, {1.0507009873554804934193349852946}));
+    auto x_f64 = context.mark_node(std::make_shared<opset8::Convert>(x, element::f64));
```

Comment:
IMHO, it is overkill to convert `x` to `f64`. I would like to refer to ONNX implementation here: https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/src/op/selu.cpp#L20-L27 where constant are created already in datatype of `x`. In our case it means applying `ConvertLike` to `alpha` and `lambda` to let constant folding make it.

Reply:
BTW, ONNX should go in the same way ideally and this code can be a part of a common translation library (that we are about to start building), but it is less common having undefined datatypes in ONNX model in comparison to our practice in PT.

Reply:
ONNX have types of inputs known, but we do not know them in general case during translation

Reply:
So basically `ConvertLike` does the same as ONNX FE

Reply:
I see what you mean, yeah it makes sense to do it like this, but what if input is integer? I think @eaidova checked that it is possible in pytorch.

Reply:
Selu for integer types? Really? Does it have practical meaning?

Reply:
I don't think it does, but torch can do it and we can do it, why not do it?

Reply:
Just covering this case we will have a kind of ugly IR with two converts to f64 and back. I believe the plugins will reduce it to their data types in the convert precision transformation and the result will be the same but I don't like the idea of always having this bigger subgraph for not really used case in practice.